clang/include/clang/Driver/Options.td

```diff
@@ -6870,6 +6870,11 @@ defm sycl_force_inline_kernel_lambda
           NegFlag<SetFalse, [], [ClangOption, CLOption], "Disallow">,
           BothFlags<[], [ClangOption, CLOption, CC1Option], " force inline "
                     "SYCL kernels lambda in entry point">>;
+defm sycl_allow_all_features_in_constexpr
```

Comment:
As this is a `-cc1` only option, this would better fit here:  https://github.com/intel/llvm/blob/d91aca01ed1ece6fe94f12fc1d72306ed0de368a/clang/include/clang/Driver/Options.td#L8494

Reply:
Yeah, makes sense, thank you.